daos-cortx/docs/setup_robinhood_policy_engine.md

```diff
@@ -0,0 +1,221 @@
+
+* Build robinhood RPMs by running
+ 
+      `./configure`
+  
+* Install missing deps -- mycase yum install jemalloc-devel.x86_64 jemalloc.x86_64
+  
+      `make rpm`
+
+* Install and start database service
+
+      `yum install mariadb-server`
+
+      `systemctl start mariadb.service`
+
+* Creating robinhood database 
+
+      `/root/setup_robinhood/robinhood-3.1.6/scripts/rbh-config create_db <db_name>    'localhost' 'rbh_password'`
+
+  A common name for robinhood database name is 'rbh_fsname'. Write the selected password to a file only readable by 'root' (600), for example in /etc/robinhood.d/.dbpassword.
+
+  Reference is available [here.](https://github.com/cea-hpc/robinhood/wiki/v3_posix_tuto#configuration)
+
+* Create a robinhood configuration file, starting with a sample robinhood template:
+
+      `cp /root/setup_robinhood/robinhood-3.1.6/doc/templates/basic.conf /etc/robihood.d/posix.conf`
+
+* Edit the configuration file
+
+  In 'General' block, set filesystem path to be scanned by Robinhood policy engine.
+       
+       fs_path = "/mnt/src_container_mnt";
+       fs_type = fuse.daos;
+       
+  Note : Make sure you use filesystem path of your interest.
+ 
+  In 'ListManager' block, set database connection parameters:
+
+      db = rbh_fsname;
+      password_file = "/etc/robinhood.d/.dbpassword" ;
+
+  It is recommended to define your fileclasses before running the initial file system scan.
+
+  This way, you will get relevant information in 'rbh-report --class-info' report after the initial scan is completed. 3 Example file-classes are give below.
+
+      fileclass all_object {
+                definition { size > 0 }
+      }
+
+      fileclass big_object {
+                definition { size > 1MB }
+      }
+
+      fileclass small_object {
+                definition { size < 1MB}
+      }
+
+
+  Reference config file is available [here.](https://github.com/Seagate/cortx-experiments/blob/main/daos-cortx/src/samples/rh_daos_cortx.conf)
+  
+  Make relevant changes in fields mentioned above.
+
+* Start Robinhood scan and update
+
+      `/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/robinhood --scan --once -L stderr -f /etc/robinhood.d/rh_daos_cortx.conf`
+
+* Query database
+ 
+      `/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/rbh-report --fs-info --class-info -f /etc/robinhood.d/rh_daos_cortx.conf`
+ 
+# Test for daos to CORTX and vice-versa data movement
+
+In this exercise object movements from daos to cortx and cortx to daos will be made sure. This exercise will be carried out by defining policies inside a container and running those policies using robinhood policy engine. There are two prerequisites to carry out this test mentioned below.
+
+Node : There's a demonstration video available for the following exercise [here](). //TODO
```

Comment:
When will this be fixed?

Reply:
I'm on it. It will get tracked [here](https://github.com/Seagate/cortx-experiments/issues/70) . Thanks John.